sqlite/writer/writer.go

```diff
@@ -261,49 +235,88 @@ func resolvePlaces(ctx *context,
 	return placeToDCID, nil
 }
 
+func prepareDatabase(fileDir string) error {
+	dbPath := path.Join(fileDir, "datacommons.db")
```

Comment:
https://stackoverflow.com/questions/39181790/whats-the-difference-between-path-path-filepath-packages-in-go

Better to use filepath.Join?

Reply:
good to know. Fixed